services/QuillLMS/spec/models/question_spec.rb

```diff
@@ -189,7 +189,8 @@
     end
 
     it 'should remove the specified incorrectSequence if stored in an array' do
-      question.update_incorrect_sequences([{'text'=>'text', 'feedback'=>'bar'}, {'text'=>'text', 'feedback'=>'bar'}, {'text'=>'text', 'feedback'=>'bar'}])
+      data = [{'text'=>'text', 'feedback'=>'bar'}, {'text'=>'text', 'feedback'=>'bar'}, {'text'=>'text', 'feedback'=>'bar'}]
+      question.data['incorrectSequences'] = data
```

Comment:
Why did these need to be rewritten to stop using the `update_incorrect_sequences` function?

Reply:
`update_incorrect_sequences` now adds a UID if there isn't one, so the data no longer matches in the test expectation. That said, I could manually add UIDs to the data here instead or have the matcher ignore the UIDs.

Reply:
this was the easiest change, but probably adding the UIDs is more correct.